framework/data/Sort.php

```diff
@@ -229,7 +229,7 @@ public function getOrders($recalculate = false)
                     $orders[$name] = $dir;
                 }
             } else {
-                $orders[] = $columns;
+                $orders = $columns;
```

Comment:
changing the return type may break code that relies on current behavior. SqlDataProvider does expect an array for example: https://github.com/yiisoft/yii2/blob/4081a4b20349974ea9f37957b166fa4398cf3a8b/framework/data/SqlDataProvider.php#L121-L126

please elaborate on why you think this change should be made. It looks wrong to me.